app/cp_nigeria/helpers.py

```diff
@@ -396,10 +396,14 @@ def get_community_region(project):
         "Ondo": "South West",
         "Lagos": "South West",
     }
-    geolocator = Nominatim(user_agent="cp_nigeria_app")
-    location = geolocator.reverse(f"{project.latitude}, {project.longitude}")
-    state = location.raw["address"]["state"]
-    region = state_region_mapping[state]
+    try:
+        geolocator = Nominatim(user_agent="cp_nigeria_app")
+        location = geolocator.reverse(f"{project.latitude}, {project.longitude}")
+        state = location.raw["address"]["state"]
+        region = state_region_mapping[state]
+    except:
```

Comment:
Can you be more specific with the Error type? Bare `except` blocks are less helpful as explicit ones